de/px-platform.json

```diff
@@ -520,5 +527,20 @@
                 }
             }
         }
+    },
+    "“izPanel”.”studiesDashboard”": {
+        "scheduledStudyTitle": "Du hast {{studiesAvailable}} verfügbare Studien und {{scheduled}} geplante"
+    },
+    "izPanel.demographicsUpdate.complete.title": {
```

Comment:
@jcagigal-uz where is this coming from? izPanel is at the root already, this seems wrong

Reply:
Hmm, this is strange, I didn't add these